ui/app/AppLayouts/Profile/views/SettingsContentBase.qml

```diff
@@ -112,19 +115,7 @@ Item {
 
                 Column {
                     id: contentWrapper
-                }
-            }
-
-            Item {
-                // This is a settingsDirtyToastMessage placeholder
-                width: settingsDirtyToastMessage.implicitWidth
-                height: settingsDirtyToastMessage.active ? settingsDirtyToastMessage.implicitHeight : 0
-
-                Behavior on implicitHeight {
-                    NumberAnimation {
-                        duration: 150
-                        easing.type: Easing.InOutQuad
-                    }
```

Comment:
Why is this? 
This was a special placeholder so that the toast don't overlap the bottom content in the scrollview.

Reply:
Now we have the bubble floating on top of the contents